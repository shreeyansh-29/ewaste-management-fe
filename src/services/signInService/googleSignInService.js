import api from "../../core/utilities/httpProvider";

export const googleSignInService = (data) => {
  return api.post(`http://localhost:8083/signin/google?email=${data.payload}`);
};
