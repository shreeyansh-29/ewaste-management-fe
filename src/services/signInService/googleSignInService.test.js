/* eslint-disable no-undef */
import React from "react";
import {googleSignInService} from "./googleSignInService";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing googleSignInService", () => {
  const data = {payload: {}};
  const Service = googleSignInService;

  const wrapper = shallow(<Service {...data} />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
