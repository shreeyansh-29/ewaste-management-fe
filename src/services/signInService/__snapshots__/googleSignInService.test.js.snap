// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing googleSignInService 1`] = `<Component />`;
